Sources/App/configure.swift

```diff
@@ -12,14 +12,21 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
 
     logger.log("👟 Starting up...")
 
-    guard let githubWebhookSecret = Environment.githubWebhookSecret
-        else { throw ConfigurationError.missingConfiguration(message: "💥 GitHub Webhook Secret is missing")}
+    services.register(logger, as: LoggerProtocol.self)
 
-    let gitHubService = GitHubService(signatureToken: githubWebhookSecret)
+    services.register(GitHubService.self) { container -> GitHubService in
+
+        logger.log("⏳ Initializing `GitHubService`...")
+
+        guard let githubWebhookSecret = Environment.githubWebhookSecret
+            else { throw ConfigurationError.missingConfiguration(message: "💥 GitHub Webhook Secret is missing")}
```

Comment:
```suggestion
            else { throw ConfigurationError.missingConfiguration(message: "💥 GitHub Webhook Secret is missing") }
```

Btw we have some convenience extensions for environment variables in Stevenson as we have to use quite a few
I wonder if we should at some point consider using Stevenson as a dependency on Wall-E for GitHub service as it also needs it, but keep the rest of logic in Wall-E itself, like we do with Stevenson app target.

Reply:
I think we should analyse that option, it makes sense to reuse some of the infrastructure (at least) between them but I think we should do it after getting it running and addressing the issues that we would like to get fixed as soon as possible.